src/os/newstore/NewStore.cc

```diff
@@ -3299,8 +3327,8 @@ int NewStore::_do_write_all_overlays(TransContext *txc,
         bl_next_data.substr_of(bl_next, next->second.value_offset,
```

Comment:
Is that wise to use 1 db read here to tradeoff 1 buffered disk write ?  I mean, by merging two overlay, we can only save one write in in _wal_apply, the write is cheap since it's an buffered_io which will be synced together with other writes.Db read seems more expensive.